.changeset/new-dryers-cough.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-auth-backend-module-oidc-provider': patch
+'@backstage/plugin-auth-node': patch
```

Comment:
Could you split this into two separate changesets? The current message doesn't quite apply to either of the packages.